src/hisp/bin.py

```diff
@@ -0,0 +1,182 @@
+from typing import List
+import pandas as pd
+
+
+class SubBin:
+    thickness: float
+    material: str
+    mode: str
+    dfw: bool
+    parent_bin_index: int
+    low_wetted_area: float  # m^2
+    high_wetted_area: float  # m^2
+    total_area: float  # m^2
+    f: float  # fraction of heat values in low_wetted_area from SMITER
+    # (f = H_low * low_wetted_area /(H_low * low_wetted_area + H_high * high_wetted_area))
```

Comment:
This is commenting, not really documenting. Documenting would be writing all these in docstrings. Please make an issue of this comment for later.